src/AzureServerless/deploy.msbuild

```diff
@@ -0,0 +1,107 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Deploy" ToolsVersion="4.0">
+	
+	<PropertyGroup>
+		<GXDeployFileProject>$([System.IO.Path]::GetFullPath('$(DeployFullPath)\..\..\..\..\web\$(ProjectName).gxdproj'))</GXDeployFileProject>
+	</PropertyGroup>
+	<Import Project="$(GX_PROGRAM_DIR)\GeneXus.AzureDeploy.targets"/>
+	<Import Project="$(GXDeployFileProject)"/>
+		
+	<PropertyGroup Condition="'$(GENERATOR)' == '.NET Core'">
+		<UploadSource>$(DeployFullPath)</UploadSource>
+		<DeploymentTargetBinPath>$(GX_PROGRAM_DIR)\DeploymentTargets\AzureFunctions\AzFunctionFiles\bin</DeploymentTargetBinPath>
+		<DeploymentTargetAzfunctPath>$(GX_PROGRAM_DIR)\DeploymentTargets\AzureFunctions\AzFunctionFiles\bin\.azurefunctions</DeploymentTargetAzfunctPath>
+		<DeploymentTargetRuntimesPath>$(GX_PROGRAM_DIR)\DeploymentTargets\AzureFunctions\AzFunctionFiles\bin\runtimes</DeploymentTargetRuntimesPath>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<AzureFunctionsRootDeps Include = "$(DeploymentTargetBinPath)\**\*.*" />
+		<AzureFunctionsFolderDeps Include = "$(DeploymentTargetAzfunctPath)\**\*.*" />
+		<AzureFunctionsRuntimesDeps Include = "$(DeploymentTargetRuntimesPath)\**\*.*" />
+	</ItemGroup>	
+
+	<Target Name="GenerateConfigs">
+
+		<Message Text="Generating configuration file..." Importance="high"/>
+		<GenerateAzFunctionConfigsHttp 
+
+				YamlFile = "$(AZURE_SERVERLESS_OPENAPI_FILE)"
+				DeployFullPath="$(DeployFullPath)" />
+			
+	</Target>
+	
+	<Target Name="PrepareDeploy" DependsOnTargets="GenerateConfigs" >
+	<Message Text="Adding required files to the deploy...(this may take a while)" Importance="high"/>
+
+		<ItemGroup>
+			<ConfResources Include="$(DeployFullPath)\*.svc" />
+			<ConfResources Include="$(DeployFullPath)\Images.txt" Condition="Exists('$(DeployFullPath)\Images.txt')"/>
+			<ConfResources Include="$(DeployFullPath)\log.config" Condition="Exists('$(DeployFullPath)\log.config')"/>
+			<ConfResources Include="$(DeployFullPath)\log.console.config" Condition="Exists('$(DeployFullPath)\log.console.config')"/>	
+			<ConfResources Include="$(DeployFullPath)\GeneXus.services" Condition ="Exists('$(DeployFullPath)\GeneXus.services')" />
+			<ConfResources Include="$(DeployFullPath)\functions.metadata"  />
+			<ConfResources Include="$(DeployFullPath)\appsettings.json" />
+		</ItemGroup>
+
+		<ItemGroup>
+			<Modules Include="@(Object -> '%(Module)')" />
+			<ModuleDirs Include="%(Modules.Identity)" />
+			<ModuleRelativePath Include="@(ModuleDirs->Replace('.', '\'))" />
+			<ModuleRelPathLower Include="@(ModuleRelativePath->ToLower())" />
```

Comment:
Same here. ModuleRelativePathLower for more readability. 
https://github.com/kettanaito/naming-cheatsheet

Reply:
OK